ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/TracingOptionsManager.java

```diff
@@ -203,19 +191,17 @@ private Properties getOptions(IPluginModelBase model) {
 	 * support cases when: key is split in multiple lines; key use escape
 	 * characters; comment uses ! start char
 	 */
-	private void loadComments(InputStream stream, Properties modelOptions) throws IOException {
+	private static void loadComments(InputStream stream, Properties modelOptions) throws IOException {
 		String prevComment = ""; //$NON-NLS-1$
 		try (BufferedReader bufferedReader = new BufferedReader(
-				new InputStreamReader(stream, Charset.defaultCharset()))) {
-			String line;
-			while ((line = bufferedReader.readLine()) != null) {
+				new InputStreamReader(stream, StandardCharsets.ISO_8859_1))) {
+			for (String line; (line = bufferedReader.readLine()) != null;) {
 				if (line.startsWith("#") || line.trim().isEmpty()) { //$NON-NLS-1$
 					prevComment += "\n" + line.trim(); //$NON-NLS-1$
```

Comment:
You could change `"\n"` with the system-independent version: `System.lineSeparator()`

Reply:
The comment string is later split by `\\r?\\n` in the only caller in `TracingPropertySource.getFormattedComment()`, so this does not really make a difference so I would therefore just keep it as is to make it simpler.